massa-bootstrap/src/tests/tools.rs

```diff
@@ -267,18 +269,18 @@ pub fn get_dummy_block_id(s: &str) -> BlockId {
     BlockId(Hash::compute_from(s.as_bytes()))
 }
 
-pub fn get_random_public_key() -> PublicKey {
-    let priv_key = KeyPair::generate();
+pub fn get_random_public_key(version: u64) -> PublicKey {
+    let priv_key = KeyPair::generate(version).unwrap();
```

Comment:
Could use some sort of range primitive to constrain the selected value? 

If you're interestedd in a type-theory rabit-hole, look into dependant typing/GATs